cloud/passportauth/integration/index.ts

```diff
@@ -0,0 +1,33 @@
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
+import * as express from 'express';
+import {PassportAuth, PassportSetup, UserApiService, UserSecService} from '../src/index';
+import {UserDataSource} from './UserDataSource';
+import userSeedData, {User} from './UserSeedData';
+
+const userSource = new UserDataSource<User>(userSeedData);
+const userApi = new UserApiService<User>(userSource);
+const userSec = new UserSecService<User>(userApi);
+const passportSetup = new PassportSetup<User>(userSec);
+const authService = new PassportAuth<User>(userSec);
+const app = express();
+
+app.use(bodyParser.json());
+app.use(cors());
+passportSetup.init(app); // initialize passport
+
+app.get('/testSecureEndpoint', authService.protect('test', '/login'), (req: express.Request, res: express.Response) => {
```

Comment:
We may need to use what's needed to cover all requirements and options. In this particular example `/login` should be set globally and users should be always redirected to this global endpoint. I do not see any real use case where app will serve 2 different login endpoints.